openstf_dbc/database.py

```diff
@@ -92,7 +95,17 @@ class DataBase:
     )
     get_random_pv_systems = _systems.get_random_pv_systems
 
-    def __init__(self):
-        """Init the stuff that also performs actions on init"""
+    def __init__(self, config=None):
+        """Construct the DataBase singleton.
+
+        Initialize the datainterface and api. WARNING: this is a singleton class when
+        calling multiple times with a config argument no new configuration will be
+        applied.
```

Comment:
Also here it is nice to document the config input argument

Reply:
done